docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +159,10 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-automatically]]
+=== Building the OpenShift Documentation Automatically
+
+To automatically get the view of {project} documentation integrated into the OpenShift documentation, trigger the link:https://ci.centos.org/job/minishift-docs/build[minishift docs] job by specifying your repo URL in `REPO` and branch name containing your documentation changes in `BRANCH` parameters respectively. This job will perform all necessary steps including the steps mentioned in xref:manually-building-openshift-docs[Manually Building the OpenShift Documentation].
+
+At the end of the successful job, you will get message where to find the hosted {project} documentation.
```

Comment:
"receive a message informing you of where to find" or "receive a message with the location of the hosted {project} documentation".